services/frontend-v3/src/i18n/fr_CA.json

```diff
@@ -258,7 +258,7 @@
   "profile.talent.matrix.result": "Résultat de la matrice de talents",
   "profile.teams": "Nom des équipes",
   "profile.telephone": "Téléphone du travail",
-  "profile.temporary.role": "Rôle temporaire?",
+  "profile.temporary.role": "Agissez-vous actuellement?",
```

Comment:
This means "are you acting/moving right now?" (Like the literal meaning) it's funny.

Maybe "Avez-vous un rôle intérimaire présentement?"